doc/nrf/app_dev/device_guides/nrf54l/fota_update.rst

```diff
@@ -0,0 +1,241 @@
+.. _ug_nrf54l_developing_ble_fota:
+
+FOTA updates on nRF54L Series devices
+#####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. fota_upgrades_intro_start
+
+|fota_upgrades_def|
+You can also use FOTA updates to replace the application.
+See the :ref:`app_dfu` page for general Device Firmware Update (DFU) information, such as supported methods for sending and receiving updates on the device.
+
+.. note::
+   For more information about introducing immutable MCUboot bootloader, refer to :ref:`ug_bootloader_adding_sysbuild_immutable_mcuboot`.
+
+.. fota_upgrades_intro_end
+
+.. _ug_nrf54l_developing_ble_fota_steps:
+
+FOTA over Bluetooth Low Energy
+******************************
+
+.. fota_upgrades_over_ble_intro_start
+
+FOTA updates are supported using MCUmgr's Simple Management Protocol (SMP) over Bluetooth®.
+The application acts as a GATT server and allows the connected Bluetooth Central device to perform a firmware update.
+To use FOTA over Bluetooth LE, samples must support Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`).
+
+The application supports SMP handlers related to:
```

Comment:
The application supports or "application should" to support, meaning is this optional for users to do for the purpose of following this guide? Can you please clarify? Otherwise this sentence fragment is confusing.

Reply:
1st and 2nd are must, 3rd is should.

Reply:
Ok. Then is there something wrong here? I followed the guide to add FOTA support to "peripheral_lbs" sample and I didn't have to add any of these handlers and the whole thing still worked. Which is why I think none of these are needed.

Reply:
Application supports them by default configuration.

Reply:
Then this should be written as:
```suggestion
By virtue of default configuration, the application supports SMP handlers related to:
```

Otherwise, to me, this reads as an instruction or a pre-requisite that makes me wonder what am I supposed to do to make application support SMP handlers.

Reply:
I'd suggest simplifying, added a suggestion

Reply:
I used suggestions from Annna.